bitrise.yml

```diff
@@ -2774,13 +2774,21 @@ workflows:
 
             brew install jq
 
-            curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -e -r '.[].commit.author | select(.name | contains("BITRISE.YML-bot"))' > /dev/null/
-            test $? -eq 0 || exit 1
+            resp=$(curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -r '.[].commit.author | select(.name | contains("BITRISE.YML-bot"))')
+            echo $resp
+            if [ -z $resp ]
```

Comment:
I think you need to wrap $resp in quotes as it will throw a ```[: too many arguments``` from https://stackoverflow.com/questions/13781216/meaning-of-too-many-arguments-error-from-if-square-brackets

```Bash
if [ -z "$resp" ]
```

Reply:
This is working as it is, I tried both with empty resp (see [build](https://app.bitrise.io/build/2720b700e45762a9#?tab=log)) and with my name as committer (see [build](https://app.bitrise.io/build/2604ce2a8919202d#?tab=log), red due to running older xcode but steps running as defined by run_if) and both seemed to work...not sure I understand what would be the problem...

@AaronMT Let me know if that it is not what you mean please...

Reply:
It might depend on the environment. It's one of those things if it happens we'll fix it. 🤷🏼‍♂️

Reply:
I will modify this on Monday to wrap it as you suggests on Monday and we can land then along with the other PR

Reply:
Ok